doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md

```diff
@@ -53,6 +86,12 @@ To install, proceed as follows:
    sudo ln -s /usr/local/include/botan-1.11/botan /usr/local/include/botan
    ```
 
+5. Make sure that dynamic loader will find the libraries in `/usr/local/lib`:
+
+   ```
+   export LD_LIBRARY_PATH=/usr/local/lib:${LD_LIBRARY_PATH}
```

Comment:
Yes, ldconfig has system-wide effect including already running processes and environments.
And the 'export' way affects only the environment (process) this was done in and is dropped with its exit.